src/ui/components/shared/WorkspaceSettingsModal/OrganizationSettings.tsx

```diff
@@ -123,6 +123,35 @@ const OrganizationSettings = ({ workspaceId }: { workspaceId: string }) => {
           </label>
         </Input>
       </Row>
+      <Row>
+        <Label>Automatically Add Users</Label>
+        <Input>
+          <label className="flex items-center" htmlFor="auto_add_users">
+            <select
+              className={classNames("rounded-sm ml-0 text-sm", {
+                "bg-toolbarBackground": disabled,
+                "border-gray-300": disabled,
+              })}
+              disabled={disabled}
+              id="auto_add_users"
+              name="auto_add_users"
+              value={workspace.settings.features.user.autoJoin || 0}
+              onChange={e =>
+                updateFeature({
+                  user: {
+                    autoJoin: Number(e.currentTarget.selectedOptions.item(0)?.value) || null,
+                  },
+                })
+              }
+            >
+              <option value="0">None</option>
+              <option value="1">Viewer</option>
+              <option value="3">Developer</option>
+              <option value="131">Admin</option>
```

Comment:
what are these magical numbers?

Reply:
They map to a bitmask on the backend. Normally we use strings on the frontend and map them to the bitmask but we don't have that hook in place for `features` atm. It's rudimentary, after all ...

Reply:
@bvaughn I think the graphql API expects strings (e.g. `viewer`, `admin`)

Reply:
It does. It's unclear to me if/how the bitmask strings factor in on the frontend anymore. I guess... maybe they don't?

Reply:
I don't think we should be using the bitmask on the frontend at this point